scripts/postinstall-typegen-script.js

```diff
@@ -15,9 +15,7 @@ async function main() {
     process.exit(0)
   }, 10 * 1000)
 
-  // npm uses __dirname as CWD in a postinstall hook. We use INIT_CWD instead if available (set by npm)
-  // https://github.com/npm/npm/issues/16990#issuecomment-349731142
-  const cwd = process.env.INIT_CWD || process.cwd()
+  const cwd = process.cwd()
```

Comment:
Are we sure that `process.cwd` function will respect the current state of `process.chdir`?

Reply:
Or is the point that it doesn't and that's what we want?

Bottom line is we want this to be the project root right?

Reply:
Yes we're sure otherwise it wouldn't have fixed the deps warning!